[21.05]  nixUnstable: 2.4pre20210922 -> 2.4pre20211006 ; init lowdown-0-9

sternenseemann opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

Backport of #139234.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

sternenseemann committed:
nixUnstable: 2.4pre20210922 -> 2.4pre20211001

Now depends on lowdown >= 0.9.

(cherry picked from commit 25da0b3bbbeb703285a493c0773dcccf3d46009b)

sternenseemann committed:
lowdown-0-9: init at 0.9.0

Backport 6ea6a232cc62269866b5690094015fcab2184e18 as a separate
attribute to save us from a breaking change (newer version needed
by nixUnstable).

r-rmcgibbo commented:
Result of `nixpkgs-review pr 140522` at a8dc6be3 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built successfully:</summary>
  <ul>
    <li>hydra-unstable</li>
    <li>lowdown-0-9</li>
    <li>nix-update</li>
    <li>nixUnstable</li>
    <li>nixpkgs-review</li>
    <li>rnix-lsp</li>
  </ul>
</details>

trofi committed:
nixUnstable: pre20211001 -> pre20211006

The specific reason to update is to pull in logging fix:
  c6718a9d950214 "Don't reset the logger in a vfork"

Otherwise 'nix build' does not report build progress correctly.

(cherry picked from commit 5323bc0f144890c23915e8d54d996a5cff97a05c)
